remote/src/main/scala/org/apache/pekko/remote/PhiAccrualFailureDetector.scala

```diff
@@ -153,16 +152,10 @@ class PhiAccrualFailureDetector(
         val interval = timestamp - latestTimestamp
         // don't use the first heartbeat after failure for the history, since a long pause will skew the stats
         if (isAvailable(timestamp)) {
-          if (interval >= (acceptableHeartbeatPauseMillis / 3 * 2) && eventStream.isDefined)
-            eventStream.get.publish(
-              Warning(
-                this.toString,
-                getClass,
-                s"heartbeat interval is growing too large for address $address: $interval millis",
-                Logging.emptyMDC,
-                RemoteLogMarker.failureDetectorGrowing(address)))
-          oldState.history :+ interval
-        } else oldState.history
+          recordInterval(interval)
```

Comment:
nit: Can we remove the unnecessary `{` and `}` in the `if`/`else` statement so its closer to the older style.

Reply:
will change it back